addons/account/models/account_lock_exception.py

```diff
@@ -326,3 +328,58 @@ def action_show_audit_trail_during_exception(self):
             'search_view_id': [self.env.ref('account.view_message_tree_audit_log_search').id],
             'domain': self._get_audit_trail_during_exception_domain(),
         }
+
+    def _get_moves_during_exception_domain(self):
```

Comment:
It would make more sense to update `_get_audit_trail_during_exception_domain`, and rename in master if you want to; I don't see any reason to keep dead code.

Reply:
I changed it as you suggest it. But does it not technically break stable policy? Since the returned domain is for a different model? It probably does not matter in practice since I doubt someone customised it already.